qt/python/mantidqt/project/project.py

```diff
@@ -13,11 +13,13 @@
 from qtpy.QtWidgets import QFileDialog, QMessageBox
 
 from mantid.api import AnalysisDataService, AnalysisDataServiceObserver
+from mantid.kernel import ConfigService
 from mantidqt.io import open_a_file_dialog
 from mantidqt.project.projectloader import ProjectLoader
 from mantidqt.project.projectsaver import ProjectSaver
 
 
+# noinspection PyTypeChecker
```

Comment:
Why are we turning off the PyTypeChecker?

Reply:
I have no idea, I don't think I put that in.